pkg/virtctl/vmexport/vmexport_test.go

```diff
@@ -254,6 +254,27 @@ var _ = Describe("vmexport", func() {
 			Expect(err.Error()).Should(ContainSubstring(expectedError))
 		})
 
+		It("VirtualMachineExport download fails when readiness timeout", func() {
+			testInit(defaultHandler)
+			vme := utils.VMExportSpecPVC(vmexportName, metav1.NamespaceDefault, "test-pvc", secretName)
+			vme.Status = &exportv1.VirtualMachineExportStatus{
+				Phase: exportv1.Pending,
+			}
+			utils.HandleVMExportGet(vmExportClient, vme, vmexportName)
```

Comment:
Please base your changes on the PR I mentioned and try to make proper use of the fake client. The `utils.*` helpers will be dropped soon.

Reply:
Thanks for the extra context, are you asking that I rebase my PR against your fork? Doing so will combine both our changes, and your code not being merged into this repository yet, I can't update the PR to diff between your fork and mine right now
If your fork has to go first, I guess I can also wait until your changes are there? It will be minor test updates anyway

Reply:
Wait for #12939 to merge to main and rebase your branch on that. You could of course temporarily rebase on @0xFelix's branch though if that will help get the code written